libvast/src/system/index.cpp

```diff
@@ -531,6 +531,10 @@ index(index_actor::stateful_pointer<index_state> self,
       } else {
         VAST_ASSERT(active.capacity >= x.rows());
         active.capacity -= x.rows();
+        VAST_TRACE(self, "reduced active partition capacity to",
+                   (std::to_string(active.capacity) + '/'
+                    + std::to_string(self->state.partition_capacity)),
+                   "rows");
```

Comment:
This message has been removed on master in the meantime but a rebase brought it back. Imho, the message has no value. Please drop that commit.